src/extensions/queue-repeat/README.md

```
@@ -0,0 +1,84 @@
+![](Foto.png)
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)